pages/api/update-a-visit.js

```diff
@@ -104,7 +104,11 @@ export default withContainer(
             : scheduledCall.recipientNumber
         );
 
-        if (!notificationType) return;
+        if (!notificationType)
+          return {
+            success: null,
```

Comment:
It feels a bit weird to use null here to imply success below (checking `emailSuccess === false` instead of `!emailSuccess`).  If `notificationType` is false, that means that the callTime and the contactMethod have not changed. As such should we return `success: true` instead of `success: null` ?